### PR TITLE
nyxt: update to 2.2.3

### PR DESCRIPTION
**Pull request by egorenar**
Signed-off-by: Alexander Egorenkov <egorenar-dev@posteo.net>

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/nyxt/template [hunk 1]
```diff
@@ -12,14 +11,19 @@ short_desc="Keyboard-oriented, extensible web-browser"
...
 # Disable check because ASDF/USER::PROVE is not installed
 make_check=no
 nostrip=yes
 nopie=yes
 nocross="Cross-compilation fails due to package iolib/syscalls setting incorrect compiler flags"
 
+do_extract() {
+	mkdir -p ${wrksrc}
+	bsdtar xvf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${pkgname}-${version}-source-with-submodules.tar.xz -C ${wrksrc}
+}
```
Comment (paper42) [inline srcpkgs/nyxt/template:25]: See https://github.com/void-linux/void-packages/pull/33871/files#r742124959

**Review by paper42 — COMMENTED** [review]
A PR for an update to 2.2.2 is in #33871, does 2.2.3 fix the issue described there?

File: srcpkgs/nyxt/template (hunk 1 above)
Comment (egorenar) [reply srcpkgs/nyxt/template:25]: Yes, it is the same issue i solved with do_extract().
Both, 2.2.2 and 2.2.3 need this fix.

Comment (paper42) [reply srcpkgs/nyxt/template:25]: > Set create_wrksrc instead.

It should be possible to delete do_extract() and replace it with create_wrksrc and wrksrc.

@WilliamBehrens could you take a quick look at this PR?

Comment (egorenar) [reply srcpkgs/nyxt/template:25]: yes, `create_wrksrc` works.

**Pull request merged**